### ISSUE TITLE
Subject: Clarification on render.py Output in Research Work

### ISSUE THREAD

**Issue by sunbeam-217** [Chaphlagical/Deblur-GS#13]
Dear Chen,Liu，

I hope this message finds you well. I am currently working with your research and have a question regarding the results produced by render.py.

Could you please confirm if the images in the **train** folder generated by render.py are the results of the deblurring process, and if the images in the **test** folder correspond to new view synthesis results?

Thank you very much for your time and assistance.

Best regards

**Comment by Chaphlagical** [comment]
Yes, the **train** folder is the deblurring results of the training images and the **test** folder is the synthesis results from novel views. By the way, make sure you use the command line parameter `--eval` so that it could split the train dataset and test dataset, or it will use all images for training.
